common.blocks/input/input.post.css

```diff
@@ -1,4 +1,3 @@
-.input
-{
+:block(input) {
```

Comment:
Can we somehow extend selectors with file context?
E.g. we can write here just `:block` and pass in entity name (or parse file name) to postcss plugin.